src/room/CallEndedView.module.css

```diff
@@ -17,20 +17,31 @@ limitations under the License.
 .headline {
   text-align: center;
   margin-bottom: 60px;
+  white-space: pre;
 }
 
 .callEndedContent {
   text-align: center;
-  max-width: 360px;
+  max-width: 450px;
+}
+.callEndedContent p {
+  font-size: 18px;
```

Comment:
Let's use one of these font size variables: https://github.com/vector-im/element-call/blob/15350b6989fee1b2e458994400daec8c4f4e6c82/src/index.css#L33